api.js

```diff
@@ -10,6 +11,7 @@ config.cwd = process.__dirname;
 
```

Comment:
Yeah we'll no longer need to change the initialization flow. I think if we keep everything to be initialized via environment variables, we won't need any strange async initialization.